src/components/requests/features.zoom.vue

```diff
@@ -14,14 +14,26 @@
       <button id="btnZoomToFeaturesRequest" class="btn btn-primary exampleready" @click="zoomToFeaturesRequest">MapModulePlugin.ZoomToFeaturesRequest</button>
     </div>
     <code-component snippet="channel.postRequest('MapModulePlugin.ZoomToFeaturesRequest',[]);"></code-component>
+
     <div>
       Zoom to features, where featureType=='parcel' and
       layerId=='testLayer'
       <button id="btnZoomToFeaturesRequest2" class="btn btn-primary exampleready" @click="ZoomToFeaturesRequestWithParams">MapModulePlugin.ZoomToFeaturesRequest</button>
     </div>
     <code-component snippet="var layers = {'layer': ['testLayer']};
-    var features = {'species': ['parcel']};
-    channel.postRequest('MapModulePlugin.ZoomToFeaturesRequest',[layers, features]);"></code-component>
+  var features = {'species': ['parcel']};
+  channel.postRequest('MapModulePlugin.ZoomToFeaturesRequest',[layers, features]);"></code-component>
+
+    <div>
+      Zoom with maxZoomLevel set
+      <button id="btnZoomToFeaturesRequest3" class="btn btn-primary exampleready" @click="zoomToFeaturesRequestMaxZoomLevel">MapModulePlugin.ZoomToFeaturesRequest</button>
+    </div>
+    <code-component snippet="var maxZoomParams = {
+      'maxZoomLevel': 4
+  };
+  channel.postRequest('MapModulePlugin.ZoomToFeaturesRequest',[maxZoomParams]);">
+    </code-component>
```

Comment:
Try if this works:
```suggestion
    <code-component>
  var maxZoomParams = {
      'maxZoomLevel': 4
  };
  channel.postRequest('MapModulePlugin.ZoomToFeaturesRequest',[maxZoomParams]);
    </code-component>
```
It should and it would be much easier to read.